### PR TITLE
Eradicate Trio references

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##

This PR removes all remaining references to Trio email addresses and replaces them with something more appropriate.

## 💭 Motivation and context ##

We are winding down our usage of Trio, so we must clean up these old references.

## 🧪 Testing ##

Tested via visual inspection.  👀 

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: config/gitleaks.toml [hunk 1]
```diff
@@ -30,7 +30,7 @@ title = "gitleaks config"
 	tags = ["email"]
 	[rules.allowlist]
 		regexes = ['''(?i)@(github.com|test.org)''',
-			'''(?i)@(trio.dhs.gov|cisa.dhs.gov|beta.dhs.gov|hq.dhs.gov)''',
+			'''(?i)@(beta|cisa|gwe.cisa|hq|trio).dhs.gov''',
```
Comment (jsf9k) [inline config/gitleaks.toml:33]: Do we need to escape periods here so they don't match any ol' character?

**Review by jsf9k — APPROVED** [review]
Approved, but please have a look at my one comment.

File: config/gitleaks.toml (hunk 1 above)
Comment (dav3r) [reply config/gitleaks.toml:33]: Since we weren't doing it before, I'm hesitant to change it now because I don't have time to test it out.

**Review by jasonodoom — APPROVED** [review]

File: config/gitleaks.toml (hunk 1 above)
Comment (jsf9k) [reply config/gitleaks.toml:33]: See #92.

**Pull request merged**